src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -1403,8 +1406,12 @@ public static Path copy(Path source, Path target, CopyOption... options)
      *          if the array contains a copy option that is not supported
      * @throws  FileAlreadyExistsException
      *          if the target file exists but cannot be replaced because the
-     *          {@code REPLACE_EXISTING} option is not specified <i>(optional
-     *          specific exception)</i>
+     *          {@code REPLACE_EXISTING} option is <i>not</i> specified,
+     *          or if the {@code REPLACE_EXISTING} option <i>is</i> specified,
+     *          the target file does not exist, the move is not an atomic file
+     *          system operation, and a file is created at the target location
+     *          between the existence check and actually moving the source
+     *          <i>(optional specific exceptions)</i>
```

Comment:
I think it might be better to split into two sentences. The second sentence could be "It may also be thrown when the REPLACE_EXISTING option is specified, the move is not atomic, and the target file is created by some other entity at around the same time that this method is called". Would that work for you?

Reply:
So changed in 06732ccbdc4f0e39434eeee51b9a44c2d5a4b6b0.